parsec/core/types/pki.py

```diff
@@ -0,0 +1,9 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
+
+from parsec._parsec import LocalPendingEnrollment, PkiEnrollmentLocalPendingError, X509Certificate
+
+__all__ = (
+    "LocalPendingEnrollment",
+    "PkiEnrollmentLocalPendingError",
+    "X509Certificate",
+)
```

Comment:
Do we really want to do that ? I thought that we want to import everything from `parsec._parsec` :thinking: